src/CMakeLists.txt

```
@@ -1619,11 +1619,16 @@ target_link_libraries(
     $<$<AND:$<BOOL:${CP2K_USE_FFTW3_}>,$<BOOL:${CP2K_ENABLE_FFTW3_OPENMP_SUPPORT}>>:cp2k::FFTW3::fftw3_omp>
     $<$<BOOL:${CP2K_USE_SPLA}>:SPLA::spla>
     $<$<BOOL:${CP2K_USE_LIBINT2}>:cp2k::Libint2::int2>
-    $<$<BOOL:${CP2K_USE_LIBTORCH}>:${TORCH_LIBRARIES}>
+    $<$<BOOL:${CP2K_USE_TREXIO}>:cp2k::trexio::trexio>
+    $<$<BOOL:${CP2K_USE_DEEPMD}>:DeepMD::deepmd>
+    $<$<BOOL:${CP2K_USE_GRPP}>:cp2k::grpp::grpp>
+    $<$<BOOL:${CP2K_USE_TORCH}>:${TORCH_LIBRARIES}>
```

This seems to re-introduce an issue I fixed in https://github.com/cp2k/cp2k/pull/3873.
```suggestion
    $<$<BOOL:${CP2K_USE_LIBTORCH}>:${TORCH_LIBRARIES}>
```